templates/header.hbs

```diff
@@ -1,4 +1,5 @@
 <a class="skip-navigation" tabindex="1" href="#main-content">{{t 'skip_navigation' }}</a>
+<a class="skip-to-web-widget" tabindex="1" href="#launcher" >{{t 'skip_to_web_widget'}}</a>
```

Comment:
In the demo video, I'm noticing that the "Skip to Main Content" button is in title case, while the "skip to web widget" button is all lowercase. Would you mind updating the Web Widget text to title case, so that it's consistent?

Reply:
Relatedly: do our customers (and our customers' customers) think of the messenger as Web Widget? Or as Messenger? We should use whatever terminology they're familiar with and/or expecting.

Reply:
Thanks  a lot Jen for reviewing. I actually changed the string to [skip to Chat ](https://github.com/zendesk/help_center/blob/main/config/locales/translations/help_center.en.yml#L6822).  The other button has [Skip to main content](https://github.com/zendesk/help_center/blob/main/config/locales/translations/help_center.en.yml#L6814C22-L6814C22). The Chat is with capital case since it's not supposed to be translated.

Reply:
Sounds good! Thanks for clarifying. 🙂